packages/react-native-sdk/src/components/call/CallParticipantsList.tsx

```diff
@@ -137,15 +137,15 @@ export const CallParticipantsList = (props: CallParticipantsListProps) => {
 
   const renderItem = useCallback<NonNullable<FlatListProps['renderItem']>>(
     ({ item: participant }) => {
-      const isVisible = viewableParticipantSessionIds.current.has(
+      const isVideoVisible = viewableParticipantSessionIds.current.has(
         participant.sessionId,
       );
       return (
-        <Participant
+        <ParticipantView
           participant={participant}
           containerStyle={itemContainerStyle}
-          kind="video"
-          isVisible={isVisible}
+          videoMode="video"
+          muteVideo={!isVideoVisible}
```

Comment:
I see that `isVisible` has been changed to `muteVideo`

but why? what was the reason.. aligns to which platform here

Reply:
Basically, android has no such prop, I completely agree. 
React has a prop named, `muteAudio` that only plays video, and audio is muted.
On a similar note, we have `muteVideo` that can be used to only play audio even when the video is off.

Since the convention that react follows is with the mute suffix I changed `isVisible` to `muteVideo`. Makes it more readable.

Reply:
Also, there is some visibility state logic in the useEffect of VideoRenderer(before ParticipantView) that revolves around this variable(before `isVisible` and now `muteVideo`). Initially, I thought of removing this variable once and for all but then I was not sure about the logic.